### PR TITLE
Remove TODO expr in filters

### PR DESCRIPTION
**Pull request by sfc-gh-jhilgart**
Currently, we're adding default filters that include a blank `expr`. However, during validation this returns an error. Now, the default filter will pass validation, but not add value until a user updates it.

### CODE REVIEW

File: semantic_model_generator/tests/generate_model_test.py [hunk 1]
```diff
@@ -314,7 +314,7 @@ def mock_dependencies_exceed_context(mock_snowflake_connection):
...
+    want_yaml = "name: this is the best semantic model ever\ntables:\n  - name: ALIAS\n    description: some table comment\n    base_table:\n      database: TEST_DB\n      schema: SCHEMA_TEST\n      table: ALIAS\n    filters:\n      - name: '  '\n        synonyms:\n          - '  '\n        description: '  '\n    dimensions:\n      - name: ZIP_CODE\n        synonyms:\n          - '  '\n        description: some column comment\n        expr: ZIP_CODE\n        data_type: TEXT\n    time_dimensions:\n      - name: BAD_ALIAS\n        synonyms:\n          - '  '\n        description: '  '\n        expr: BAD_ALIAS\n        data_type: TIMESTAMP\n    measures:\n      - name: AREA_CODE\n        synonyms:\n          - '  '\n        description: '  '\n        expr: AREA_CODE\n        data_type: NUMBER\n      - name: CBSA\n        synonyms:\n          - '  '\n        description: '  '\n        expr: CBSA\n        data_type: NUMBER\n"
```
Comment (sfc-gh-rehuang) [inline semantic_model_generator/tests/generate_model_test.py:317]: oh, the expr is also auto-populated for filters. 
I thought this model'll still be broken without expr for filters?

Comment (sfc-gh-jhilgart) [reply semantic_model_generator/tests/generate_model_test.py:317]: You're right. Looking

Comment (sfc-gh-rehuang) [reply semantic_model_generator/tests/generate_model_test.py:317]: one thought maybe we mark the whole filter area as comments.... and let user know you can uncomment if you want to add filter

Comment (sfc-gh-jhilgart) [reply semantic_model_generator/tests/generate_model_test.py:317]: I thought this was the issue, but turns out this is a valid filter.

```
    filters:
      - name: '  ' # <FILL-OUT>
        synonyms:
          - '  ' # <FILL-OUT>
        description: '  ' # <FILL-OUT>
        expr: '  ' # <FILL-OUT>
```

And we already include the col name in the expr (so we are generating valid yaml in this generator).

**Pull request closed without merging**